src/wwwroot/src/model/i18n/cz.ts

```diff
@@ -0,0 +1,390 @@
+/*!
+ * Bravo for Power BI
+ * Copyright (c) SQLBI corp. - All rights reserved.
+ * https://www.sqlbi.com
+*/
+import { Locale } from '../i18n';
+import { strings as _ } from '../strings';
+
+const locale: Locale = {
+
+    locale: "en", //DO NOT TRANSLATE
+    name: "English", //DO NOT TRANSLATE
```

Comment:
Should be `name: "Czech"`
Also, can you please remove `//DO NOT TRANSLATE` as this is a misleading comment from the original source file.